src/main/java/ru/sbtqa/tag/pagefactory/PageFactory.java

```diff
@@ -83,6 +93,46 @@ public static void initElements(FieldDecorator decorator, Object page) {
     public static PageWrapper getInstance() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 30 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3776)